chaintest/action_test_helpers.go

```diff
@@ -71,6 +72,11 @@ func (test *ActionTest) Run(ctx context.Context, t *testing.T) {
 	t.Run(test.Name, func(t *testing.T) {
 		require := require.New(t)
 
+		for _, act := range test.SetupActions {
+			_, err := act.Execute(ctx, test.Rules, test.State, test.Timestamp, test.Actor, test.ActionID)
+			require.NoError(err)
+		}
+
```

Comment:
Is it necessary to separate between `SetupActions` and `Action`? Could we just change `Action` to a slice of actions and execute them sequentially?

Reply:
I am definitely for that! From DevX pov, I thought it would be better to label these actions as `SetupActions` since their correctness isn't being tested (whereas `Action` is being tested for correctness). However, we can just make `Action` an array and make the developer responsible for its usage (via reading the documentation).